app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentModule.kt

```diff
@@ -103,7 +104,8 @@ class EnrollmentModule(
             schedulerProvider,
             formRepository,
             valueStore,
-            analyticsHelper
+            analyticsHelper,
+            context.getString(R.string.field_is_mandatory)
```

Comment:
do we have any content provider to gather all this access to string resources?

Reply:
Nope... but i think it is a great idea. @JaimeToca I'm going to create a Quality ticket

Reply:
Hi @andresmr do you have in mind like a repository or datasource that receives and id an fetch de string? If so, I think this takes 10 sec

Reply:
In fact, i think we have the resourceManager class to get the icons form the sdk. It could be extended

Reply:
I was thinking in a provider where yo can ask for explicit content decoupling the ids from the Modules classes and presenters etc... for example `contentProvider.provideFieldMandatoryMessage()` and you'll receive the String properly formatted and localized. What do you think?

Reply:
sounds good to me, this should go in a SDK PR, right?

Reply:
Nope, this is app side